modules/grafana/Makefile

```diff
@@ -0,0 +1,43 @@
+# You might want to set these when you run make
+GRAFANA_ADMIN_PASSWORD ?= admin
+LOCAL_DASHBOARD_DIRECTORY ?= dashboards
+
+# You should leave these as the default values
+GRAFANA_LOCAL_DOCKER_NAME = grafana_local
+GRAFANA_SYNC_DOCKER_NAME = grafana_sync
+CONTAINER_DASHBOARD_DIRECTORY = /app/dashboards
+TMP_GITLAB_REPO_DIRECTORY = /tmp/build-harness-extentions-private
+
+.PHONY: grafana/develop
+# Start up a local grafana instance with live datasources, and sync to dashboards on your local disk. Only for mintel internal access, try grafana/develop-opensource for general use.
+grafana/develop: grafana/setup-local-grafana-mintel grafana/setup-grafana-syncer grafana/cleanup
+
+.PHONY: grafana/develop-opensource
+# Start up a local grafana instance with no datasources configured, and sync to dashboards on your local disk
+grafana/develop-opensource: grafana/setup-local-grafana-opensource grafana/setup-grafana-syncer grafana/cleanup
```

Comment:
Might consider a rename to `grafana/develop-oss` (just quicker to type)

Reply:
Done in new revision